pyflp/plugin.py

```diff
@@ -67,6 +68,23 @@ class _FruityBalanceStruct(StructBase):
     PROPS = {"pan": "I", "volume": "I"}
 
 
+class _FruityBloodOverdriveStruct(StructBase):
+    PROPS = dict.fromkeys(
+        (
+            "plugin_marker",
+            "pre_band",
+            "color",
+            "pre_amp",
+            "x100",
+            "post_filter",
+            "post_gain",
+            "unknown_event_first",
+            "unknown_event_second",
+        ),
+        "I",
+    )
+
+
```

Comment:
Note:

To support both the recent and the old version of BloodOverdrive two Structs would be used here.

`class _FruityBloodOverdriveStruct(StructBase):`
-> 
`class _FruityBloodOverdriveNewStruct(StructBase):`

```
class _FruityBloodOverdriveOldStruct(StructBase):
    PROPS = dict.fromkeys(
        (
            ...
            "pre_band",
            "color",
            "pre_amp",
            "x100",
            "post_filter",
            "post_gain",
            ...
        ),
        "I",
    )
```

Reply:
Didn't think about how big the names would get, use `_FruityBloodOverdriveStruct2` probably?